tests/backends/plotly/plotly_canvas_test.py

```diff
@@ -7,6 +7,8 @@
 import tempfile
 import os
 
+pytest.importorskip("plotly")
```

Comment:
Do you have (non-release) tests that try to run tests without plotly? Otherwise I can see it coming: We will reintroduce imports in some of the plotly modules and a release build will fail because of this. I think we either need to add such a test or move the `importorskip` to the top.

Reply:
moving the `importorskip` to the top does not help, because it just doesn't run the tests, but it still tries to import the plotly Canvas just after the `importorskip` :-(

Reply:
But I agree with your concern, that this only runs during a release build.

Reply:
Can you make another tox env that does not include plotly?

Reply:
See update